examples/developer_guide/README.md

```diff
@@ -61,11 +62,16 @@ Once a new release has been made, and its corresponding version tag exists on th
   ```
 After running the above, create a PR for the changes that the script made.
 
-Once your changes have been merged, similar to the [Deployment](#deploying) section above, before deleting/leaving your branch, update
+Once your changes have been merged, similar to the deploying section above, before deleting/leaving your branch, update
 the example images to use the merged commit SHA:
 ```shell
 ./.hack/update_examples.sh -us <commit-sha>
 ./.hack/update_examples.sh -bp
 ```
 As a result, the correct SDK version will always be printed in the server information logs, and
 the example images will always be using the latest commit SHA.
+
+### Adding a New Example
+
+If you add a new example, in order for it to be used by the Docker Publish workflow, add its path
+to the `dockerfile_paths` matrix in `build-push.yaml`.
```

Comment:
Do we have a Docker Publish workflow for Python? do you mean updating the script?